onnxruntime/contrib_ops/cuda/bert/embed_layer_norm_impl.cu

```diff
@@ -193,13 +194,15 @@ bool LaunchEmbedLayerNormKernel(
     return EmbedSkipLayerNorm<half>(
         stream, hidden_size, batch_size, sequence_length, input_ids, segment_ids,
         reinterpret_cast<const half*>(beta), reinterpret_cast<const half*>(gamma),
-        reinterpret_cast<const half*>(word_embedding), reinterpret_cast<const half*>(position_embedding), reinterpret_cast<const half*>(segment_embedding),
+        reinterpret_cast<const half*>(word_embedding), reinterpret_cast<const half*>(position_embedding), 
+        reinterpret_cast<const half*>(segment_embedding), epsilon,
```

Comment:
Here epsilon is cast from double to half. Note that the minimum positive half (2^-24 is around 6e-8). Need we to do some handling to make sure epsilon is positive after cast.